layouts/shortcodes/blocks/resource.html

```diff
@@ -10,7 +10,7 @@
 {{ $title := .Get "title" | default "Video title" }}
 
 <div class="col-lg-4 mb-3 mb-lg-0 text-center ">
-  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" src="{{ $thumbnail }}"></a>
+  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" alt="video" src="{{ $thumbnail }}"></a>
```

Comment:
```suggestion
  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" alt="{{ $title | markdownify }}" src="{{ $thumbnail }}"></a>
```

how about this? can you try it and see if it works for you? maybe add a `title` attribute as well?

Reply:
Done